aries-site/src/examples/templates/forms/SignInExample.js

```diff
@@ -0,0 +1,175 @@
+  const [formValues, setFormValues] = React.useState({ resetEmail: email });
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your password reset logic here
+    // Display success status
+    closeLayer();
+  };
+
+  return (
+    <>
+      <Box
+        direction="row"
+        justify="end"
+        pad={{ horizontal: 'small', top: 'small' }}
+      >
+        <Button
+          a11yTitle="Close reset password form"
+          icon={<Close />}
+          onClick={closeLayer}
+        />
+      </Box>
+      <Box
+        gap="medium"
+        margin={{ horizontal: 'xlarge', bottom: 'xlarge', top: 'large' }}
+        width="medium"
+      >
+        <Heading level={2} margin="none">
+          Reset Password
+        </Heading>
+        <Form
+          validate="blur"
+          value={formValues}
+          onChange={setFormValues}
+          onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+        >
+          <Box gap="medium">
+            <Text>
+              An email to reset your password will be sent to the following
+              address:
+            </Text>
+            <FormField
+              name="resetEmail"
+              component={TextInput}
+              type="email"
+              placeholder="your.email@company.com"
+              validate={emailValidation}
+            />
+            <Button label="Send Password Reset" primary type="submit" />
+          </Box>
+        </Form>
+      </Box>
+    </>
+  );
+};
+
+export const SignInExample = () => {
+  const [formValues, setFormValues] = React.useState({});
+  const [showForgotPassword, setShowForgotPassword] = React.useState(false);
+
+  const onClose = () => {
+    setShowForgotPassword(false);
+  };
+
+  const onForgotPassword = () => {
+    setShowForgotPassword(true);
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Sign In
+          </Heading>
+          <Text>to Hewlett Packard Enterprise</Text>
+        </Header>
+        <Main
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            validate="blur"
+            value={formValues}
+            onChange={setFormValues}
+            onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          >
+            <FormField
+              name="email"
+              label="Email"
+              component={TextInput}
+              placeholder="james@hpe.com"
+              type="email"
+              validate={emailValidation}
+            />
+            <FormField
```

Comment:
I think this is a grommet bug, but when I run accessibility check it says there is no associated label with the formfield.

Reply:
Can you remind me which accessibility tool you are running? Would it make sense to add it to our pre-commit checks?

Reply:
It could be done with axe if we loop through all the routes on the site, then every page will be checked for accessibility but I didn't know if that would add too much time to the tests. If you want to just to a test as you develop, you can add WAVE chrome browser extension

Reply:
Gotcha. Thank you.